src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. If the main one doesn't work, the browser will use the alternative one. For example: `font-family: Roboto, Arial, sans-serif;`